platform-api/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/server/model/impl/CommandImpl.java

```diff
@@ -65,26 +65,28 @@ public void setType(String type) {
     }
 
     @Override
-    public boolean equals(Object obj) {
-        if (this == obj) {
-            return true;
-        }
-        if (!(obj instanceof CommandImpl)) {
-            return false;
-        }
-        final CommandImpl command = (CommandImpl)obj;
-        return Objects.equals(name, command.name) &&
-               Objects.equals(commandLine, command.commandLine) &&
-               Objects.equals(type, command.type);
+    public String getPreviewUrl() {
+        return previewUrl;
+    }
+
+    public void setPreviewUrl(String previewUrl) {
+        this.previewUrl = previewUrl;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        CommandImpl command = (CommandImpl)o;
+        return com.google.common.base.Objects.equal(getName(), command.getName()) &&
```

Comment:
It's just autoreplacement during generating equals through the Intellij IDEA. Will fix that.